nicegui/elements/scene_object3d.py

```diff
@@ -42,16 +42,19 @@ def with_name(self, name: str) -> Self:
         self._name()
         return self
 
-    def send(self) -> None:
-        """Send the object to the client."""
-        self._create()
-        self._name()
-        self._material()
-        self._move()
-        self._rotate()
-        self._scale()
-        self._visible()
-        self._draggable()
+    @property
+    def data(self) -> List[Any]:
```

Comment:
I would use a simple method, not a property.

Reply:
Why do you think a method would be better?